deployments/tools/export/lakefs_export.py

```diff
@@ -0,0 +1,117 @@
+import subprocess
+import argparse
+import os
+import shlex
+import time
+from datetime import datetime
+from string import Template
+
+LAKEFS_ACCESS_KEY = os.getenv('LAKEFS_ACCESS_KEY')
+LAKEFS_SECRET_KEY = os.getenv('LAKEFS_SECRET_KEY')
+LAKEFS_ENDPOINT = os.getenv('LAKEFS_ENDPOINT')
+S3_ACCESS_KEY = os.getenv('S3_ACCESS_KEY')
+S3_SECRET_KEY = os.getenv('S3_SECRET_KEY')
+
+SUCCESS_MSG = "Export completed successfully!"
+
+
+def create_rclone_conf_file():
+    with open('rclone.conf.template') as f:
+        src = Template(f.read())
+        vars = {'lakefs_access_key': LAKEFS_ACCESS_KEY, 'lakefs_secret_key': LAKEFS_SECRET_KEY, 'lakefs_endpoint': LAKEFS_ENDPOINT, 's3_access_key': S3_ACCESS_KEY, 's3_secret_key': S3_SECRET_KEY}
+        res = src.substitute(vars)
+    with open('rclone.conf', 'w+') as f:
+        f.write(res)
+    return
+
+
+def set_args():
+    parser = argparse.ArgumentParser(description='Process lakeFS export.')
+
+    parser.add_argument('Repo', metavar='repo', type=str, help='the name of lakeFS repository')
+    parser.add_argument('Dest', metavar='dest', type=str, help='path of destination')
+    parser.add_argument('--branch', metavar='branch', type=str, action='store', help='the relevant branch in the repository to export from')
+    parser.add_argument('--commit_id', metavar='commit', type=str, action='store', help='the relevant commit on the repository to export from')
```

Comment:
I don't understand the difference between `commit_id` and `branch`.  They are both examples of what we call "refs".  And there are many very useful refs that are not commit IDs or branches, such at _tags_ (think "export version `1.1.4` of my data) and relative names (think "export `main^`, the previous version of `main`).

I would just unify two into a single arg, unless it makes a difference.  You can add examples to the help message to make it clear what you can pass in, e.g.
```
"ref to export from repository, e.g. `main', `v1.0.1`, `main~3`"
```

Reply:
You'r very right. But this is how the Spark export works. Other refs are not supported.. I can open an issue for that and we'll add it to both.